docs/user-guide/cli-using-using-team-profiles.md

```diff
@@ -54,10 +80,8 @@ You can continue using Zowe CLI V1 profiles with Zowe CLI V2. However, we highly
    
 :::
 
-
 - Commands in the `zowe config` [command group](../user-guide/cli-using-understanding-core-command-groups#config) now let you manage security for any option value.
 
 - The `zowe scs` and `zowe config` command groups were repurposed to work with team profiles.
```

Comment:
@adam-wolfe @zFernand0 Is this accurate? Elsewhere we say that `zowe scs` commands are obsolete. Which is correct?

Reply:
The secure credential store plug-in (that contributes the scs command group) doesn't really work with Zowe V2. I would just delete this sentence.

Reply:
I agree with Adam, it's probably best to remove it 😋